tensorflow_datasets/image/fruits360_test.py

```diff
@@ -0,0 +1,38 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for fruits360 dataset module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import fruits360
+
+num_classes = len(fruits360._CLASS_NAMES)
+
+
+class FruitsTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = fruits360.Fruits360
+
+  SPLITS = {
+    "train": 6,
+    "test": 6,
+  }
+
```

Comment:
Also need the S3 test.

Reply:
@ChanchalKumarMaji https://www.tensorflow.org/datasets/add_dataset#testing_mydataset should be updated with the S3 test.